apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLoggerFactoryExtensions.cs

```diff
@@ -33,17 +33,31 @@ public static class GoogleLoggerFactoryExtensions
         public static ILoggerFactory AddGoogle(this ILoggerFactory factory, string projectId,
             LoggerOptions options = null, LoggingServiceV2Client client = null)
         {
-            // Check params and set defaults if unset.
             GaxPreconditions.CheckNotNull(factory, nameof(factory));
```

Comment:
This null check is no longer needed since it is being done in the other overload.